Documentation for improved wildcard search (#3725)
SoftwareAG/c8y-docs: content/cockpit/using-widgets-bundle/selecting-assets-widgets.md

There are two methods which can be used to quickly find assets:

> 1. Name search, and

Reviewer: ```suggestion
1. Name search
```

> 2. filtering.

Reviewer: ```suggestion
2. Filtering
```

#### Name search {#name-search}

On the **Configuration** tab of the widget editor you can use the search field under **Asset selection**.

> Through the search you can find assets in the whole hierarchy, but it requires the name to include the search term.

Reviewer 2: ```suggestion
Through the search you can find assets anywhere within the hierarchy, but the asset’s name must contain the search term.
```

> Through the search you can find assets anywhere within the hierarchy, but the asset’s name must contain the search term.

Reviewer: ```suggestion
With the search, you can find assets anywhere within the hierarchy if the asset’s name contains the search term.
```
